conductor_api_api/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
@AshantiMutinta, take note of this new crate. It only includes this `ConductorApi` struct which is a proxy for the conductor api that does not depend on anything in the `conductor_api` crate - which is good because we can't depend on that in the `net` crate without creating cyclic dependencies. `ConductorApi` was in `core` before we can't depend on that either.

With the repository clean-up that we discussed yesterday, I would suggest with rename this crate to `conductor_api` since the current `conductor_api` will be `conductor_lib`, right?